bpf/lib/common.h

```diff
@@ -220,13 +221,21 @@ struct lb4_service {
 	__u16 port;
 	__u16 count;
 	__u16 rev_nat_index;
+	__u16 weight;
 } __attribute__((packed));
 
 struct lb4_reverse_nat {
 	__be32 address;
 	__u16 port;
 } __attribute__((packed));
 
+/* Must match lbmap.go maxSeq */
+#define LB_RR_MAX_SEQ 31
+struct lb_sequence {
+	__u16 count;
+	__u16 idx[LB_RR_MAX_SEQ];
```

Comment:
No I cannot do that for constants. The struct size needs to be known at compile time. 
May be the makefile approach is the only way to set constants. Something like 
go build $(GOBUILD) -ldflags "-X "github.com/cilium/cilium/bpf/lbmap".maxSeq=$(CILIUM_LB_RR_MAX_SEQ)" -o $(TARGET) ./main.go
I need to try it out. This is basically the struct and we are trying to set maxSeq externally.

package main

import "fmt"
import "unsafe"

const (
        maxSeq = 31
)

type RRSeqValue struct {
        Count uint16
        Idx   [maxSeq]uint16
}

func main() {
        fmt.Printf("%d\n", int(unsafe.Sizeof(RRSeqValue{})))
}